ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1266,6 +1266,12 @@ void GCS_MAVLINK::handleMessage(mavlink_message_t* msg)
 
         switch(packet.command) {
 
+        case MAV_CMD_DO_CHANGE_SPEED:
+            AP_Mission::Mission_Command cmd;
+            AP_Mission::mavlink_cmd_long_to_mission_cmd(packet, cmd);
+            plane.do_change_speed(cmd);
+            break;
```

Comment:
Also, if it mavlink_cmd_long_to_mission_cmd does not return accepted then you should not call plane.do_change_speed()